[2023-03-01 12:45:57,087] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-11T00:05:00+00:00 [queued]>
[2023-03-01 12:45:57,322] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-11T00:05:00+00:00 [queued]>
[2023-03-01 12:45:57,334] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:45:57,335] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 12:45:57,337] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:45:57,605] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-11-11T00:05:00+00:00
[2023-03-01 12:45:57,630] {standard_task_runner.py:52} INFO - Started process 12441 to run task
[2023-03-01 12:45:57,687] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-11-11T00:05:00+00:00', '--job-id', '2901', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmp72o9p0ig', '--error-file', '/tmp/tmp1ssqsho_']
[2023-03-01 12:45:57,699] {standard_task_runner.py:77} INFO - Job 2901: Subtask get_name
[2023-03-01 12:45:58,088] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-11-11T00:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 12:45:58,801] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-11-11T00:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-11T00:05:00+00:00
[2023-03-01 12:45:58,998] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 12:45:59,105] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221111T000500, start_date=20230301T124557, end_date=20230301T124559
[2023-03-01 12:45:59,828] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 12:45:59,951] {local_task_job.py:146} INFO - Task exited with return code 0
